src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
shape and color have defined possible values. let's create types for them (enum or union)